### PR TITLE
Womtool graph for draft 3

### PR DESCRIPTION
**Pull request by cjllanwarne**
As side quests, this included:

* No longer requiring inputs in order to build a `WomBundle` (what `wom graph` really wants, and what imports should really produce).
* A second step to go from `WomBundle` to `WomExecutable`/`ValidatedWomNamespace`
* Fixing draft 3 input processing in the `LanguageFactory`.

### CODE REVIEW

File: languageFactories/wdl-draft2/src/main/scala/languages/wdl/draft2/WdlDraft2LanguageFactory.scala [hunk 1]
```diff
@@ -77,14 +83,16 @@ class WdlDraft2LanguageFactory() extends LanguageFactory {
...
       _ <- validateWorkflowNameLengths(wdlNamespace)
       importedUris = evaluateImports(wdlNamespace)
-      womExecutable <- wdlNamespace.toWomExecutable(Option(source.inputsJson))
+      womExecutable <- wdlNamespace.toWomExecutable(Option(source.inputsJson)) // wdlNamespace.toWomExecutable(Option(source.inputsJson))
       validatedWomNamespaceBeforeMetadata <- LanguageFactoryUtil.validateWomNamespace(womExecutable)
       _ <- checkTypes(wdlNamespace, validatedWomNamespaceBeforeMetadata.womValueInputs)
-    } yield validatedWomNamespaceBeforeMetadata.copy(importedFileContent = importedUris)).toValidated
+    } yield validatedWomNamespaceBeforeMetadata.copy(importedFileContent = importedUris)
+
+    val errorOr: ErrorOr[ValidatedWomNamespace] = checked.toValidated
```
Comment (mcovarr) [inline languageFactories/wdl-draft2/src/main/scala/languages/wdl/draft2/WdlDraft2LanguageFactory.scala]: ew

File: languageFactories/wdl-draft2/src/main/scala/languages/wdl/draft2/WdlDraft2LanguageFactory.scala [hunk 2]
```diff
@@ -77,14 +83,16 @@ class WdlDraft2LanguageFactory() extends LanguageFactory {
       collectImportEvents
     }
 
-    val errorOr: ErrorOr[ValidatedWomNamespace] = (for {
+    val checked: Checked[ValidatedWomNamespace] = for {
       wdlNamespace <- wdlNamespaceValidation.toEither
       _ <- validateWorkflowNameLengths(wdlNamespace)
       importedUris = evaluateImports(wdlNamespace)
-      womExecutable <- wdlNamespace.toWomExecutable(Option(source.inputsJson))
+      womExecutable <- wdlNamespace.toWomExecutable(Option(source.inputsJson)) // wdlNamespace.toWomExecutable(Option(source.inputsJson))
```
[the remark this answers is not in the document]
Comment (cjllanwarne) [reply languageFactories/wdl-draft2/src/main/scala/languages/wdl/draft2/WdlDraft2LanguageFactory.scala]: This is just me taking "self-commenting code" to its logical conclusions...

File: centaur/src/main/resources/standardTestCases/wdl_draft3/passthrough_value/passthrough_value.inputs.json [hunk 3]
```diff
@@ -0,0 +1,3 @@
+{
+  "passthrough_value.x": 25
+}
```
[the remark this answers is not in the document]
Comment (cjllanwarne) [reply centaur/src/main/resources/standardTestCases/wdl_draft3/passthrough_value/passthrough_value.inputs.json]: Exactly - this works now so I might as well ensure that it doesn't regress.

File: languageFactories/wdl-draft2/src/main/scala/languages/wdl/draft2/WdlDraft2LanguageFactory.scala (hunk 1 above)
Comment (cjllanwarne) [reply languageFactories/wdl-draft2/src/main/scala/languages/wdl/draft2/WdlDraft2LanguageFactory.scala]: And then straight back `.toCheck`... yeah I think I can improve this

**Pull request merged**